lib/integrations/getIntegrations.ts

```diff
@@ -72,6 +73,14 @@ export const ALL_INTEGRATIONS = [
     description: "Video Conferencing",
     variant: "conferencing",
   },
+  {
+    installed: true,
```

Comment:
We should check for the env variables here to consider it installed. We should think of self-hosted instances that don't want/need every integration.

Reply:
Ah, thanks.  I hardcoded it while developing & forgot to circle back.